.github/CODEOWNERS

```diff
@@ -0,0 +1,233 @@
+/.github/ @elastisys/goto-pipeline/qa/release
```

Comment:
This doesn't seem to be allowed. Or is it just a bad parser?

Also, do we want to have sub teams with this instead of having it as a parent group? Then the goto-pipeline would automatically include all in all the subteams